client/components/mma/accountoverview/AccountOverview.tsx

```diff
@@ -121,6 +121,7 @@ const AccountOverviewRenderer = ([mdapiObject, cancelledProductsResponse]: [
 							{activeProductsInCategory.map((productDetail) =>
 								featureSwitches.accountOverviewNewLayout ? (
 									<AccountOverviewCardV2
+										data-qm-masking="blocklist"
```

Comment:
As above, this is being set as a prop on a React component so won't appear in the rendered HTML.

Reply:
Same comment as above